aepp/configs.py

```diff
@@ -128,15 +136,15 @@ def importConfigFile(
         }
         if sandbox is not None: ## overriding sandbox from parameter
             args["sandbox"] = sandbox
-
         if auth_type == "jwt":
             args["tech_id"] = provided_config["tech_id"]
             args["path_to_key"] = provided_config["pathToKey"]
-        elif auth_type == "oauth":
+        elif auth_type == "oauthV2":
+            args["scopes"] = provided_config["scopes"].replace(' ','')
```

Comment:
This is because when you copy the scope from the developer.adobe.com, it provides you the value with a space after the comma. (no kidding)
So if people use the copy button or doesn't clean them it maintain the space. 
I am not sure if supported or not but in the CURL command that is exposed, there are no space, so prefer to stay consistent with the CURL command.